examples/plot_04_tracking_items.py

```diff
@@ -0,0 +1,161 @@
+"""
+.. _example_historization_items:
+
+==================================
+Tracking items using their history
+==================================
+
+This example illustrates how skore can be used to track some items using their history,
+for example tracking some ML metrics over time.
+"""
+
+# %%
+# Creating and loading the skore project
+# ======================================
+
+# %%
+# We start by creating a temporary directory to store our project such that we can
+# easily clean it after executing this example. If you want to keep the project,
+# you have to skip this section.
+import tempfile
+from pathlib import Path
+
+temp_dir = tempfile.TemporaryDirectory(prefix="skore_example_")
+temp_dir_path = Path(temp_dir.name)
+
+# %%
+import subprocess
+
+# create the skore project
+subprocess.run(
+    f"python3 -m skore create my_project --working-dir {temp_dir.name}".split()
+)
```

Comment:
please use `skore.create` after merge of https://github.com/probabl-ai/skore/pull/744

Reply:
Done